src/main/java/core/basesyntax/model/Account.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+public class Account {
+    private int bananaBalance = 0;
+    private int appleBalance = 0;
```

Comment:
non extensible approach
let's follow structure outlined in the task description
create a model for Fruit itself, which accepts string as a name - you won't need any new fields when new fruit appears